src/main/java/com/es/phoneshop/model/product/ArrayListProductDao.java

```diff
@@ -68,8 +120,7 @@ public void delete(Long id) throws NoSuchElementException {
                 products.remove(
                         products.stream()
```

Comment:
Move to the separate method

Reply:
why? method delete is one of the main in DAO concept

Reply:
Move to the separate method logic of searching product to remove